### ISSUE TITLE
stack expects each tensor to be equal size, but got [1, 960, 728] at entry 0 and [2, 960, 728] at entry 10

### ISSUE THREAD

**Issue by onlyjokers** [ZHO-ZHO-ZHO/ComfyUI-YoloWorld-EfficientSAM#50]
Error occurred when executing Yoloworld_ESAM_Zho:

stack expects each tensor to be equal size, but got [1, 960, 728] at entry 0 and [2, 960, 728] at entry 10

File "D:\SD\ComfyUI_windows_portable_nvidia_cu121_or_cpu\ComfyUI_windows_portable\ComfyUI\execution.py", line 151, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
File "D:\SD\ComfyUI_windows_portable_nvidia_cu121_or_cpu\ComfyUI_windows_portable\ComfyUI\execution.py", line 81, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
File "D:\SD\ComfyUI_windows_portable_nvidia_cu121_or_cpu\ComfyUI_windows_portable\ComfyUI\execution.py", line 74, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
File "D:\SD\ComfyUI_windows_portable_nvidia_cu121_or_cpu\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-YoloWorld-EfficientSAM\YOLO_WORLD_EfficientSAM.py", line 194, in yoloworld_esam_image
new_masks = torch.stack(processed_masks, dim=0)

**Comment by rossaai** [comment]
Try a confidence threshold of 0.00 or 0.01 https://github.com/ltdrdata/ComfyUI-Impact-Pack/issues/468#issuecomment-1954328230
